src/GridFieldOrderableRows.php

```diff
@@ -17,7 +17,7 @@ class GridFieldOrderableRows extends BaseGridFieldOrderableRows
     protected function reorderItems($list, array $values, array $order)
```

Comment:
Should the params also have `[]` instead of `array`? Also add `DataList` to `$list`